common/lib/xmodule/xmodule/js/src/xmodule.js

```diff
@@ -78,7 +78,7 @@
         }
 
         try {
-            module = new window[moduleType](element);
+            module = new window[moduleType](element, runtime);
```

Comment:
@mikix @MatthewPiatetsky This seems to have caused a regression for the Conditional Block. After cherry-picking it, the `conditional_get` handler is not being called causing the ConditionalBlock to display empty. This is because `calledElId` [here](https://github.com/edx/edx-platform/blob/ca6914c/common/lib/xmodule/xmodule/js/src/conditional/display.js#L9) gets set to `runtime` and then the `render()` on line 17 is not called.

See https://openedx.atlassian.net/browse/TNL-8318 for details.

Reply:
Ah geeze sorry - let me work on a fix.

Reply:
PR for fix: https://github.com/edx/edx-platform/pull/27625  (needs some more local testing, but I think that's the idea)

Reply:
Thanks! 🙂